fe/fe-core/src/main/java/org/apache/doris/qe/MasterOpExecutor.java

```diff
@@ -163,6 +179,24 @@ private void forward() throws Exception {
         }
     }
 
+    // The SetStmt needs to be executed in follower again
+    // for the current session after master success
+    private void afterForwardExecute() throws DdlException {
+        if (parsedStmt != null) {
+            if (parsedStmt instanceof SetStmt) {
+                SetExecutor executor = new SetExecutor(ctx, (SetStmt) parsedStmt);
+                try {
+                    executor.setSessionVars();
```

Comment:
I think we can set remove the `global` property in `SetStmt` and than call `executor.execute()` method. No need to create a new method `setSessionVars()`.

Reply:
> I think we can set remove the `global` property in `SetStmt` and than call `executor.execute()` method. No need to create a new method `setSessionVars()`.

Ok, I see what you mean.